_posts/2024-10-15-anaconda-defaults.md

```diff
@@ -0,0 +1,175 @@
+
+<figure class="floating">
+  <img src="{% link assets/images/way-of-the-mambalorian.webp %}" alt="In the art style of 1990s Disney, cel shading: A mysterious blacksmith in a dark, forge-like setting. She wears a sleek, gold-plated helmet embossed with the symbol of a snake and an armored robe with intricate designs. Sparks fly as she hammers a molten weapon on an anvil, which begins to take the shape of a coiled snake. The forge’s flames cast a warm glow, reflecting off her helmet and tools. The snake is a python with a green sheen.">
+  <figcaption>The way of the Mambalorian is miniforge distribution. This is the way.</figcaption>
+</figure>
+
+This has been brewing for a number of years. The first change happened [back in 2020](https://www.anaconda.com/blog/sustaining-our-stewardship-of-the-open-source-data-science-community), and the second happened very [recently in March 2024](https://legal.anaconda.com/policies/en/?name=terms-of-service#anaconda-terms-of-service) that affects ["government entities and non-profit entities with over 200 employees or contractors"](https://www.theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/). This is problematic due the wording around 'employees' - many organisations have hundreds of employees but few users of `conda` software. 
+
+In order to avoid any potential problems, avoiding `defaults` channel is the best course of action.
+
+Before we dive in, a quick recap on definitions:
+
+- [Anaconda Inc.](https://www.anaconda.com/about-us) is the commercial entity behind the `conda`, `miniconda`, and the `Anaconda.Navigator` software suite.
+- Anaconda Inc. curates a set of packages which are available [through specific channels](https://repo.anaconda.com/pkgs/) - also known as `defaults` channel  
+- [Miniforge](https://github.com/conda-forge/miniforge) is a non-Anaconda community-developed installer specific to `conda-forge` channel
+- `mamba` is a drop in replacement to `conda`.
+- Far more detailed information on all the different channels/distributions [can be found here.](https://bioconda.github.io/faqs.html#what-s-the-difference-between-anaconda-conda-miniconda-mamba-mambaforge-micromamba)
+
+All the hoohah surrounds the [curated Anaconda channels](https://docs.anaconda.com/working-with-conda/reference/default-repositories/), commonly referred to as `defaults` channel. **Note: When installing `conda/miniconda` software, the `defaults` channel is added to your global channels list.** You could inadvertently be using Anaconda services without intending to.
+
+## How to's
+
+Below is guidance on how to best deal with `defaults` channel.
+
+### Safest: Fresh Install of Miniforge distribution
+
+1. Install the conda-forge distribution `miniforge3` - [instructions here](https://github.com/conda-forge/miniforge), this will also install `mamba`.
+2. Add the channels `bioconda` and `nodefaults` [in that order] as global defaults:
+  ```
+  conda config --add channels bioconda nodefaults
```

Comment:
There isn't a description to what `nodefaults` is so far, it might be a bit confusing for some people